packages/babel-helper-builder-react-jsx-experimental/src/index.js

```diff
@@ -836,6 +836,7 @@ You can set \`throwIfNamespace: false\` to bypass this warning.`,
   function buildCreateElementOpeningElementAttributes(file, path, attribs) {
     // TODO (Babel 8): Only leave this branch of the code and remove the rest
     if (
+      process.env.BABEL_8_BREAKING ||
```

Comment:
wasn't strictly necessary here but I've thought that this might come in handy here

Reply:
Ehhh please don't do this yet :sweat_smile: 

1. This will need to be
	```js
	(process.env.BABEL_8_BREAKING ? true : false)
    ```

	I know that it looks horrible, but to avoid usages of `process.env.BABEL_8_BREAKING` that cause us not to be able to completely remove the Babel 8 code at build time, we are only allowing it to be used as `if (process.env.BABEL_8_BREAKING)`, `if (!process.env.BABEL_8_BREAKING)`, `process.env.BABEL_8_BREAKING ? :`, or `!process.env.BABEL_8_BREAKING ? :`.

2. When making this the default, we'll need to update a lot of tests passing `"BABEL_8_BREAKING": true/false`, so it's better to keep it in a separate PR.

Reply:
I've refactored this to use `(process.env.BABEL_8_BREAKING ? true : false)`, let me know if that's OK here